stable_baselines/a2c/utils.py

```diff
@@ -169,7 +169,7 @@ def apply_action_mask(input_tensor, mask_tensor, scope):
     mask_tensor
     """
     with tf.variable_scope(scope):
-        return tf.add(input_tensor, mask_tensor, name="masking")
+        return tf.multiply(tf.nn.sigmoid(input_tensor, name="mask_relu"), mask_tensor, name="masking")
```

Comment:
The makes sure valid action values are not close to 0

Reply:
Why?
input_tensor can also contain negative values and this is not a pb, no?